inference-engine/ie_bridges/python/src/openvino/inference_engine/ie_api.pyx

```diff
@@ -245,48 +245,37 @@ cdef class IECore:
         return versions
 
     ## Reads a network from the Intermediate Representation (IR) and creates an `IENetwork`.
-    #  @param model: A `.xml`, `.onnx`or `.prototxt` model file or string with IR.
-    #  @param weights: A `.bin` file of the IR. Depending on `init_from_buffer` value, can be a string path or
-    #                  bytes with file content.
-    #  @param init_from_buffer: Defines the way of how `model` and `weights` attributes are interpreted.
-    #                           If  `False`, attributes are interpreted as strings with paths to .xml and .bin files
-    #                           of IR. If `True`, they are  interpreted as Python `bytes` object with .xml and .bin files content.
+    #  @param model: A `.xml`, `.onnx`or `.prototxt` model file or a 'bytes' object which contains a model.
+    #  @param weights: A `.bin` file of the IR. Can be a string path or bytes with file content.
     #  @return An `IENetwork` object
     #
     #  Usage example:\n
     #  ```python
     #  ie = IECore()
     #  net = ie.read_network(model=path_to_xml_file, weights=path_to_bin_file)
     #  ```
-    cpdef IENetwork read_network(self, model: [str, bytes, Path], weights: [str, bytes, Path] = "", init_from_buffer: bool = False):
```

Comment:
I think we need to keep init_from_buffer flag and when it is set just print deprecation warning that it is not needed anymore. Otherwise it is not backward compatible for users who use it.

Reply:
Can we add the second method with init_from_buffer argument and deprecate it?

I mean, do something in order to this method was deprecated with standard python deprecation approach (which can be recognized as deprecated by PyCharm and other IDEs).
Or is it not possible for cython?

Reply:
@ilyachur Yes, it's impossible for cython.